### PR TITLE
JavaScript code cleanup

### PR DESCRIPTION
**Pull request by ChristianMurphy**
Used static analysis to clean out orphan code and align code style to uPortal code style.

Static anaylsis performed by [ESLint](http://eslint.org/).

Configuration used:

``` yaml
env:
  browser: true
  jquery: true
globals:
  notificationsAccordion: false
extends: 'eslint:recommended'
rules:
  indent:
    - error
    - 4
  linebreak-style:
    - error
    - unix
  quotes:
    - error
    - single
  semi:
    - error
    - always

```

### CODE REVIEW

File: notification-portlet-webapp/src/main/webapp/scripts/jquery.notice.js [hunk 1]
```diff
@@ -25,148 +25,148 @@ var upnotice = upnotice || {};
...
+            $.ajax({
+                url      : settings.getNotificationsUrl,
+                type     : 'POST',
+                dataType : 'json',
+                success: function (data) {
+                    var feed = data.feed;
+                    callback(feed);
+                },
+                error: function () {
+                    container.html(' ').text('AJAX failed. ~ THE END ~');
```
Comment (ChristianMurphy) [inline notification-portlet-webapp/src/main/webapp/scripts/jquery.notice.js]: I am unsure how to resolve this.
`container` is undefined here, and it is unclear if or where the error message should appear.

Comment (ChristianMurphy) [inline notification-portlet-webapp/src/main/webapp/scripts/jquery.notice.js]: Copying authors of file
\cc @drewwills @codepuppet @timlevett @jameswennmacher

Comment (jhelmer-unicon) [inline notification-portlet-webapp/src/main/webapp/scripts/jquery.notice.js]: Heh.  It's a pretty poor error message too.  Most end users won't have a clue to what that means.

File: notification-portlet-webapp/src/main/webapp/scripts/job-postings.js [hunk 2]
```diff
@@ -150,14 +138,14 @@ var jobPostings = function(){
...
             $('#' + settings.portletId + ' .modal-overlay').show();
         });
-        $('#' + settings.portletId + ' .cancelEmailButton').on('click', function(e) {
+        $('#' + settings.portletId + ' .cancelEmailButton').on('click', function() {
             $('#' + settings.portletId + ' .modal-overlay').hide();
         });
-        $('#' + settings.portletId + ' .sendEmailButton').on('click', function(e) {
-            $.post( "test.php", $('#' + settings.portletId + ' .emailFriendForm').serialize());
+        $('#' + settings.portletId + ' .sendEmailButton').on('click', function() {
+            $.post( 'test.php', $('#' + settings.portletId + ' .emailFriendForm').serialize());
```
Comment (ChristianMurphy) [inline notification-portlet-webapp/src/main/webapp/scripts/job-postings.js]: Its also somewhat curious that there is a call to `test.php` in a portlet with no PHP code.

**Comment by ChristianMurphy** [comment]
closing this out, when I have a chance I'll go through and apply the Google code style to the JavaScript in this project.

**Pull request closed without merging**